content/posts/numpy/numpy2/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "18 years in the Making: NumPy 2.0 Released"
```

Comment:
If you have a minute or two to think about the title, it'd be good to make it punchy, crisp, and a summary of the announcement here. First thoughts, "NumPy 2.0: 18 years of experience, distilled", "NumPy 2.0: an evolutionary milestone", etc. etc.

Reply:
Agreed, those look better - starting with "NumPy 2.0" is nicer. I'd go for one of the first two ideas here. If I get one vote, it goes to "NumPy 2.0: an evolutionary milestone"

Reply:
Yeah, those titles are nicer.  Not sure I love the "evolution milestone", but happy with it and have no other idea :).

Reply:
We can generate others: "NumPy 2.0: milestone release brings significant improvements" or "NumPy 2.0: core ecosystem package gets major overhaul".

Reply:
The last one is quite relevant i.t.o. this being published on the ecosystem blog.

Reply:
I like "NumPy 2.0: milestone release brings significant improvements" too. Or "NumPy 2.0: an evolutionary milestone" still. The ecosystem one less so - it's a NumPy blog by the NumPy team after all.

Reply:
Happy with either choice.

Reply:
Pushed "NumPy 2.0: an evolutionary milestone" since it is the one with four people happy with it, and is short/punchy/descriptive.